org.libretrainsim.community.json

```diff
@@ -0,0 +1,61 @@
+{
+	"app-id": "org.libretrainsim.community",
```

Comment:
This is a weird app ID: it make us think that the name of the application is "community".